README.adoc

```diff
@@ -78,8 +78,9 @@ MicroProfile Config combines configuration properties from multiple sources, eac
 
 A higher ordinal value means that the values taken from this ConfigSource will override values from ConfigSources with a lower ordinal value.
 
-There are three default configuration sources as following:
+There are four default configuration sources as following:
```

Comment:
I'd suggest updating this to
"The following four sources are the default configuration sources:"